src/main/java/com/auth0/client/auth/AuthAPI.java

```diff
@@ -698,4 +750,89 @@ public AuthRequest exchangeCode(String code, String redirectUri) {
         request.addParameter("redirect_uri", redirectUri);
         return request;
     }
+
+    /**
+     * Create a request to send an email containing a link or a code to begin authentication with Passwordless connections.
+     *
+     * <pre>
+     * {@code
+     * AuthAPI auth = new AuthAPI("me.auth0.com", "B3c6RYhk1v9SbIJcRIOwu62gIUGsnze", "2679NfkaBn62e6w5E8zNEzjr-yWfkaBne");
+     * try {
+     *      PasswordlessEmailResponse result = auth.startPasswordlessEmailFlow("user@domain.com", PasswordlessEmailConnection.CODE)
+     *          .execute();
+     * } catch (Auth0Exception e) {
+     *      // Something happened
+     * }
+     * }
+     * </pre>
+     *
+     * @param email the email address to send the code or link to. Must not be null.
+     * @param connection the type of the passwordless email request. Must not be null.
+     *
+     * @return a Request to configure and execute.
+     *
+     * @see <a href="https://auth0.com/docs/connections/passwordless/guides/email-otp">Passwordless Authentication with Email documentation</a>
+     * @see <a href="https://auth0.com/docs/api/authentication#get-code-or-link">Get code or link API reference documentation</a>
+     */
+    public CustomRequest<PasswordlessEmailResponse> startPasswordlessEmailFlow(String email, PasswordlessEmailConnection connection) {
+        Asserts.assertNotNull(email, "email");
+        Asserts.assertNotNull(connection, "connection");
+
+        String url = baseUrl
+                .newBuilder()
+                .addPathSegment(PATH_PASSWORDLESS)
+                .addPathSegment(PATH_START)
+                .build()
+                .toString();
+
+        CustomRequest<PasswordlessEmailResponse> request = new CustomRequest<>(client, url, "POST", new TypeReference<PasswordlessEmailResponse>() {
+        });
+        request.addParameter(KEY_CLIENT_ID, clientId);
+        request.addParameter(KEY_CLIENT_SECRET, clientSecret);
+        request.addParameter(KEY_CONNECTION, "email");
```

Comment:
Do we want to extract this "connection" value to the method signature? (without adding another overloaded method) or should we leave that edge case (pwdless connections created via the API) to be updated later via the "set parameters" method that every CustomRequest has?

Reply:
I'm not sure I follow. This method is specific to starting the password flow using email, so the connection value wouldn't change.

Reply:
Ah, I understand now, you are referring to the ability to create custom passwordless connections via the Management API. I'll look into the best way to handle this; on first glance I'm inclined to support this via overriding the parameters via `CustomRequest` for this edge case.

Reply:
>  on first glance I'm inclined to support this via overriding the parameters via CustomRequest for this edge case.
I think that's fine, given that the creation of those connections is not straightforward from the dashboard.